src/properties/peerlistwidget.cpp

```diff
@@ -170,9 +173,9 @@ void PeerListWidget::showPeerListMenu(const QPoint&)
   QAction *dlLimitAct = 0;
 #endif
   QAction *banAct = 0;
-  QAction *copyIPAct = 0;
+  QAction *copyPeersAct = 0;
   if (!selectedPeerIPs.isEmpty()) {
-    copyIPAct = menu.addAction(IconProvider::instance()->getIcon("edit-copy"), tr("Copy IP"));
+    copyPeersAct = menu.addAction(IconProvider::instance()->getIcon("edit-copy"), tr("Copy selected peers"));
```

Comment:
Other items in the menu are in lowercase.
```c++
    addPeerAct = menu.addAction(IconProvider::instance()->getIcon("user-group-new"), tr("Add a new peer..."));
    copyPeersAct = menu.addAction(IconProvider::instance()->getIcon("edit-copy"), tr("Copy selected peers"));
    dlLimitAct = menu.addAction(QIcon(":/Icons/skin/download.png"), tr("Limit download rate..."));
    upLimitAct = menu.addAction(QIcon(":/Icons/skin/seeding.png"), tr("Limit upload rate..."));
    banAct = menu.addAction(IconProvider::instance()->getIcon("user-group-delete"), tr("Ban peer permanently"));
```
About the other thing, this is the uTorrent menu:
![](http://i57.tinypic.com/1z6siv8.png)
I have no problem in doing what you say, just want to have the feature in qBittorrent.